src/db/db.module.ts

```diff
@@ -0,0 +1,49 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+interface EnvironmentVariables {
+  ENVIRONMENT: Environment;
+  DB_AUTOLOADENTITIES: boolean;
+  DB_SYNCHRONIZE: boolean;
+  DB_MIGRATIONS_RUN: boolean;
+}
```

Comment:
Maybe it's good to maintain one shared interface for the entire app, WDYT?

Reply:
you are right. I'll implement that